kolibri/core/assets/src/core-app/apiSpec.js

```diff
@@ -159,7 +158,7 @@ export default {
       AppBar,
       AppBarPage,
       ImmersivePage,
-      SidePanelModal,
+      CoreBase,
```

Comment:
`CoreBase` has been removed, you should remove this reference - seems like a bad merge conflict resolution.
```suggestion
```